src/nfa/nfa.cc

```diff
@@ -480,13 +444,12 @@ BoolVector Nfa::get_useful_states() const {
     return useful;
 }
 
-StateSet Nfa::get_useful_states_old() const
-{
+StateSet Nfa::get_useful_states_old() const {
```

Comment:
If it is old, why do we keep it? Can we fuck it out, or (if used somewhere) rename to something better?

Reply:
We probably could. However, I am not sure if we intend to do some kind of testing with this older implementation before removing it entirely. If not, it should be alright to delete it.

Reply:
I would just remove it, I don't think someone will experiment with it.

Reply:
Agreed. I will remove the function in this PR.

Reply:
Yes, I would remove it.

Reply:
remove!

Reply:
Removed.